idea-plugin/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'java'
+apply plugin: 'com.palantir.external-publish-intellij'
+
+intellij {
+    pluginName = "gradle-consistent-versions"
+    updateSinceUntilBuild = true
+    version = "2024.1"
+    plugins = ['java', 'org.jetbrains.plugins.gradle']
+}
+
+patchPluginXml {
+    pluginDescription = "Language plugin for version.props allowing for auto-lock updating"
+    version = "1.0.0" // For some reason project.version is unspeificed and thus this fails
+    sinceBuild = '241' // TODO: test against this version of IntelliJ to ensure no regressions
+    untilBuild = ''
+}
+
+afterEvaluate {
+    tasks.named('test').configure { it.javaLauncher = null }
+}
+
+// Javadoc fails if there are no public classes to javadoc, so make it stop complaining.
+tasks.named("javadoc", Javadoc) {
+    failOnError = false
+}
+
+tasks.withType(JavaCompile).configureEach {
+    options.errorprone.disable 'PreferSafeLoggableExceptions'
+    options.errorprone.disable 'PreferSafeLogger'
```

Comment:
Might want to do this [in baseline like we do here for java-gradle-plugin](https://github.com/palantir/gradle-baseline/blob/fd590105ee4f4d8e746224d680f3bf8d557ab0d0/gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java#L100-L114) (except for intellij)

Reply:
added to baseline - see https://github.com/palantir/gradle-baseline/pull/2878